test/integration/tcproute_test.go

```diff
@@ -94,55 +97,49 @@ func TestTCPRouteEssentials(t *testing.T) {
 
 	t.Logf("exposing deployment %s/%s via service", deployment1.Namespace, deployment1.Name)
 	service1 := generators.NewServiceForDeployment(deployment1, corev1.ServiceTypeLoadBalancer)
-	// we have to override the ports so that we can map the default TCP port from
-	// the Kong Gateway deployment to the tcpecho port, as this is what will be
-	// used to route the traffic at the Gateway (at the time of writing, the
-	// Kong Gateway doesn't support an API for dynamically adding these ports. The
-	// ports must be added manually to the config or ENV).
+	// Use the same port as the default TCP port from the Kong Gateway deployment
+	// to the tcpecho port, as this is what will be used to route the traffic at the Gateway.
+	const service1Port = ktfkong.DefaultTCPServicePort
 	service1.Spec.Ports = []corev1.ServicePort{{
 		Name:       "tcp",
 		Protocol:   corev1.ProtocolTCP,
-		Port:       ktfkong.DefaultTCPServicePort,
+		Port:       service1Port,
 		TargetPort: intstr.FromInt(test.EchoTCPPort),
 	}}
 	service1, err = env.Cluster().Client().CoreV1().Services(ns.Name).Create(ctx, service1, metav1.CreateOptions{})
 	assert.NoError(t, err)
 	cleaner.Add(service1)
 
 	t.Logf("exposing deployment %s/%s via service", deployment2.Namespace, deployment2.Name)
+	const service2Port = 8080 // Use a different port that listening on the Gateway for TCP.
```

Comment:
Ditto about being more descriptive

```suggestion
	const service2Port = 8080 // Use a different port than Gateway's TCP listener port.
```